### DISCUSSION TITLE
Vector "Cutter" Eraser

### DISCUSSION THREAD

**Discussion by joelmayer87** [keiwando/toonsquid#499] [Ideas]
Both Clip Studio and Toon Boom have this tool and boy it's such a timesaver. Basically what it does is to erase a stroke from a vector layer until the next cross section of a drawing. So you can just swipe over strokes to delete them basically.

This video of the Clip Studio version probably demonstrates this more clearly:

https://youtu.be/mGd1dF011oQ?si=8KGhLLkP7tfSTgqg

**Comment by Cat-Butler** [comment]
This would be appropriate to further development of Path layers (which I'd personally like to see), which are the closest equivalent to classic vectors. To my understanding, the current "vector" brushes in ToonSquid are more like auto-traced pixel brushes, so they don't have a central stroke to intersect with other strokes.

**Reply by joelmayer87** [reply]
Neither does the Toon Boom brush tool, that's just a behaving like flash and doesn't have a centerline either. Still, as long as it intersects, it creates points and therefore could be cut in theory. Mind you i'm no developer and Toon Boom developed their vector engine for decades hence it's arguably the best in the business but still, would be cool to see.

**Comment by core999** [comment]
+1 from me the cutter tool works great in Grease Pencil too.

**Comment by Lioncub901** [comment]
This will be useful.

**Comment by IndiCros** [comment]
Yes please!!!
